app/scripts/directives/nav.js

```diff
@@ -273,10 +294,47 @@ angular.module('openshiftConsole')
         };
 
         $scope.catalogLandingPageEnabled = !Constants.DISABLE_SERVICE_CATALOG_LANDING_PAGE;
-        var select = $elem.find('.selectpicker');
-        var options = [];
 
-        var updateOptions = function() {
+        var contextSelector = $elem.find('.contextselector');
+        var previousContextSelectorValue;
+        $scope.clusterConsoleURL = window.OPENSHIFT_CONSTANTS.TECTONIC_URL;
+        contextSelector
```

Comment:
I'd wrap this in an

```js
if ($scope.clusterConsoleURL) { ... }
```

so you're not initializing the select when we don't need to. Then you can also change the `ng-show` in the view to an `ng-if`.

Reply:
I believe the select initializes no matter what since initialization occurs as a result of https://silviomoreto.github.io/bootstrap-select/.  If i change to an ng-if, said initialization doesn't  occur since the <select class="selectpicker contextpicker"> isn't present in the markup.